.github/workflows/builder.yml

```
@@ -11,18 +11,16 @@ env:
   ALPINE_LATEST: "3.22"
   DEBIAN_LATEST: "trixie"
   UBUNTU_LATEST: "24.04"
-  RASPBIAN_LATEST: "trixie"
   PYTHON_LATEST: "3.13"
```

```suggestion
  PYTHON_LATEST: "3.14"
```

The default for Python should be updated to `3.14`. It was deferred previously as 3.14 was only added from `amd64` and `aarch64`.